Add security and comms info to agent docs (#238)
elastic/observability-docs: docs/en/ingest-management/faq.asciidoc

[discrete]
[[what-is-the-endpoint-package]]
== What is the {elastic-endpoint-integration} integration in {fleet}?

The {elastic-endpoint-integration} integration provides protection on your {agent}
controlled host. The integration monitors your host for security-related events,
allowing for investigation of security data through the {security-app} in {kib}.
The {elastic-endpoint-integration} integration is managed by {agent} in the

> same way as other integrations. Try it out! For more information, see the
> {security-guide}/index.html[{elastic-sec} solution documentation].

Reviewer: ```suggestion
same way as other integrations. Try it out! For more information, see the
{security-guide}/index.html[{elastic-sec} documentation].
```

[discrete]
[[how-are-security-to-agent-communications-secured]]
== How are communications secured between {elastic-sec} and {agent}?

{elastic-sec} connects to the agent over loopback TLS on port 6788.
{elastic-sec} validates that the agent has root (Linux and macOS) or SYSTEM
(Windows) permissions.

> //REVIEWERS: Unsure here whether we want to say Elastic Endpoint or Elastic

Reviewer 2: Elastic Security right @jmikell821?

Author: Janeen confirmed over  chat that this is good.

//Security. ^^

> //REVIEWERS: In 7.10, does Elastic Security validate the digital signature of

Reviewer: Hey, @ferullo - do you know the answer to this question?

Reviewer 3: It does not in 7.10.

Reviewer 3: In 7.10 Agent validates the signature of Endpoint before installing Endpoint. I don't think it validates it after that.

> //REVIEWERS: In 7.10, does Elastic Security validate the digital signature of
> // the agent and vice versa??

Author: ```suggestion
```

[discrete]
[[how-are-agent-kibana-communications-secured]]
== How are communications secured between {agent} and {kib}?

{agent} connects to {kib} over TLS and authenticates the certificate

> presented by {kib}. The agent then provides an ApiKey as an authentication

Reviewer: ```suggestion
presented by {kib}. The agent then provides an API key as an authentication
```
assuming that's what you meant here?
